examples/toolchain_v1alpha1_nstemplateset_cr.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: toolchain.dev.openshift.com/v1alpha1
 kind: NSTemplateSet
 metadata:
+  name: johnsmith-templateset
```

Comment:
I guess, for NSTemplateSet.Name as of now there is NO need to add suffix `-templateset` and we can stay with NSTemplateSet.Name = `username`.

Also check design doc - https://docs.google.com/document/d/19UzaXVqiOo9JqDYz3qUKguJwW-7zceA4UtTr358R_CA/edit#heading=h.autst8l8rdnd